### PR TITLE
Run rustfmt on liballoc_jemalloc.

### PR DESCRIPTION
**Pull request by ahmedcharles**

### CODE REVIEW

File: src/liballoc_jemalloc/lib.rs [hunk 1]
```diff
@@ -63,40 +62,52 @@ const MIN_ALIGN: usize = 8;
 const MIN_ALIGN: usize = 16;
 
 // MALLOCX_ALIGN(a) macro
-fn mallocx_align(a: usize) -> c_int { a.trailing_zeros() as c_int }
+fn mallocx_align(a: usize) -> c_int {
+    a.trailing_zeros() as c_int
+}
 
 fn align_to_flags(align: usize) -> c_int {
-    if align <= MIN_ALIGN { 0 } else { mallocx_align(align) }
```
Comment (cristicbz) [inline src/liballoc_jemalloc/lib.rs]: I'm sympathetic to single expression if-branches to go one after another, for ternary-operator-like formatting.

Comment (ahmedcharles) [inline src/liballoc_jemalloc/lib.rs]: Feel free to submit an issue or a pull request: https://github.com/nrc/rustfmt

**Pull request merged**